packages/api/src/command/medical/patient/get-patient.ts

```diff
@@ -3,12 +3,14 @@ import { Op, Transaction } from "sequelize";
 import NotFoundError from "../../../errors/not-found";
 import { FacilityModel } from "../../../models/medical/facility";
 import { OrganizationModel } from "../../../models/medical/organization";
-import { Patient, PatientData, PatientModel, splitName } from "../../../models/medical/patient";
+import { Patient, PatientData, PatientModel } from "../../../models/medical/patient";
 import { capture } from "../../../shared/notifications";
 import { Util } from "../../../shared/util";
 import { getFacilities } from "../facility/get-facility";
 import { getOrganizationOrFail } from "../organization/get-organization";
-import jaroWinkler from "jaro-winkler";
+import { calculatePatientSimilarity } from "./calculate-patient-similarity";
+
+const JW_THRESHOLD = 0.95;
```

Comment:
not to be nit-picky, but maybe `SIMILARITY_THRESHOLD` would be easier to understand for those reading this and now knowing what JW means? 😄